### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by itsjoekent**

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
    * @see https://stackoverflow.com/a/42604801/4422345 for where this code was taken from.
    */
 export const stringifyQuery = (params = {}, prefix) => {
-  const query = Object.keys(params).map((key) => {
+  const query = Object.keys(params ? params : {}).map((key) => {
```
Comment (weerd) [inline src/helpers.js]: Move this to outside the query. No need to bother continuing if there are no `params`. Makes it more explicit too.

```
if (! params) {
  return '';
}
```

**Comment by itsjoekent** [comment]
As I pointed out to Diego in Slack, it's not really clear why the Stackoverflow answer had that bloated return statement at the end of the function, so I cleaned that up a bit as well.

File: src/helpers.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
     const value  = params[key];
```
Comment (weerd) [inline src/helpers.js]: can you remove the 👾 here: `value`👾 `=`.

Comment (itsjoekent) [reply src/helpers.js]: @weerd Refresh the page haha

File: src/helpers.js [hunk 3]
```diff
@@ -6,6 +6,8 @@
    * @see https://stackoverflow.com/a/42604801/4422345 for where this code was taken from.
    */
 export const stringifyQuery = (params = {}, prefix) => {
+  if (! params) return '';
```
Comment (weerd) [inline src/helpers.js]: When we add ESLint to this it will yell at you for not using `{...}`. Use them ;)

File: src/helpers.js [hunk 4]
```diff
@@ -6,6 +6,8 @@
...
+
   const query = Object.keys(params).map((key) => {
     const value  = params[key];
```
Comment (weerd) [inline src/helpers.js]: Remove the 👾 between `value` and `=`

Comment (itsjoekent) [reply src/helpers.js]: Sneaky little 👾

File: src/helpers.js (hunk 3 above)
Comment (itsjoekent) [reply src/helpers.js]: _yells about eslint mandating brackets for one line statements_

**Review by DFurnes — APPROVED** [review]

File: src/helpers.js [hunk 5]
```diff
@@ -23,5 +27,5 @@ export const stringifyQuery = (params = {}, prefix) => {
     return `${key}=${encodeURIComponent(value)}`;
   });
 
-  return [].concat.apply([], query).join('&');
+  return query.join('&');
```
Comment (weerd) [inline src/helpers.js:30]: Out of curiosity I was investigating, and from what I can tell, in the `RestApiClient.js` file, where `url.search = stringifyQuery(query);` is called, I'm guessing assigning the string to the `url.search` property automagically prepends the `?` to the string?

Comment (DFurnes) [reply src/helpers.js:30]: Yeah, this seems a little confusing. It looks like the setter for `.search` adds the leading `?` if you don't provide it, but the getter always returns the full query string (e.g. `?page=2`)

**Pull request merged**